project/Dependencies.scala

```diff
@@ -15,7 +15,9 @@
  * limitations under the License.
  */
 
-import sbt._
+import sbt.*
+import sbt.Def
+import sbtunidoc.GenJavadocPlugin.autoImport.unidocGenjavadocVersion
```

Comment:
Do we actually nee these changes? We don't use `sbt.Def` in this class, and I think `unidocGenjavadocVersion` should be autoimported and not needed. And the `sbt.*` import isn't standard scala is it? This feels like an auto-importer gone wrong.

Reply:
Well I get clear errors if I don't have the import of unidocGenjavadocVersion. It has no type information for that object otherwise.

Reply:
Yep, you're right, I guess SBT *.scala files don't work like .sbt files.